tests/DependencyInjection/SonataAdminExtensionTest.php

```diff
@@ -65,208 +84,81 @@ public function testHasSecurityRoleParameters()
 
-        $this->assertEquals($stylesheets, [
-            'bundles/sonatacore/vendor/bootstrap/dist/css/bootstrap.min.css',
-            'bundles/sonatacore/vendor/components-font-awesome/css/font-awesome.min.css',
-            'bundles/sonatacore/vendor/ionicons/css/ionicons.min.css',
-            'bundles/sonataadmin/vendor/admin-lte/dist/css/AdminLTE.min.css',
-            'bundles/sonataadmin/vendor/iCheck/skins/square/blue.css',
-            'bundles/sonatacore/vendor/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css',
-            'bundles/sonatacore/vendor/select2/select2.css',
-            'bundles/sonatacore/vendor/select2-bootstrap-css/select2-bootstrap.min.css',
-            'bundles/sonataadmin/vendor/x-editable/dist/bootstrap3-editable/css/bootstrap-editable.css',
-            'bundles/sonataadmin/css/styles.css',
-            'bundles/sonataadmin/css/layout.css',
-            'bundles/sonataadmin/css/tree.css',
-            'foo/bar.css',
-            'bar/quux.css',
-        ]);
+        $this->assertSame(array_merge(array_diff($this->defaultStylesheets, $removeStylesheets), $extraStylesheets), $stylesheets);
 
-        $this->assertEquals($javascripts, [
-            'bundles/sonatacore/vendor/jquery/dist/jquery.min.js',
-            'bundles/sonataadmin/vendor/jquery.scrollTo/jquery.scrollTo.min.js',
-            'bundles/sonatacore/vendor/moment/min/moment.min.js',
-            'bundles/sonatacore/vendor/bootstrap/dist/js/bootstrap.min.js',
-            'bundles/sonataadmin/vendor/jqueryui/ui/minified/jquery-ui.min.js',
-            'bundles/sonataadmin/vendor/jqueryui/ui/minified/i18n/jquery-ui-i18n.min.js',
-            'bundles/sonatacore/vendor/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js',
-            'bundles/sonataadmin/vendor/jquery-form/jquery.form.js',
-            'bundles/sonataadmin/vendor/x-editable/dist/bootstrap3-editable/js/bootstrap-editable.min.js',
-            'bundles/sonatacore/vendor/select2/select2.min.js',
-            'bundles/sonataadmin/vendor/admin-lte/dist/js/app.min.js',
-            'bundles/sonataadmin/vendor/iCheck/icheck.min.js',
-            'bundles/sonataadmin/vendor/slimScroll/jquery.slimscroll.min.js',
-            'bundles/sonataadmin/vendor/waypoints/lib/jquery.waypoints.min.js',
-            'bundles/sonataadmin/vendor/waypoints/lib/shortcuts/sticky.min.js',
-            'bundles/sonataadmin/vendor/masonry/dist/masonry.pkgd.min.js',
-            'bundles/sonataadmin/Admin.js',
-            'bundles/sonataadmin/treeview.js',
-            'bundles/sonataadmin/sidebar.js',
-            'foo/bar.js',
-            'bar/quux.js',
-        ]);
+        $javascripts = $this->container->getDefinition('sonata.admin.pool')->getArgument(3)['javascripts'];
+
+        $this->assertSame(array_merge(array_diff($this->defaultJavascripts, $removeJavascripts), $extraJavascripts), $javascripts);
```

Comment:
It seems that his line is over 120 characters, could you please break it? Thank you